static/js/typing_data.js

```diff
@@ -21,8 +21,7 @@ function get_key(group) {
     return ids.join(",");
 }
 
-export function add_typist(group, typist) {
-    const key = get_key(group);
+function update_typist_dct(key, typist) {
```

Comment:
This refactor to the `update_typist_dct` interface would be nice if extracted as a prep commit.

I'd also call this `set_typing_state(key, typist)`.

Reply:
`add_typist` works for both if `key` is extracted out. So we don't need this refactor now!